packages/playground/src/components/manage_gateway_dialog.vue

```diff
@@ -411,7 +413,11 @@ export default {
 
         await deployGatewayName(grid, selectionDetails.value!.domain, gwConfig);
         suggestName();
-        layout.value.setStatus("success", "Successfully deployed gateway.");
+        // get gateway url
+        await loadGateways();
```

Comment:
```suggestion
        const gatewayUrl = selectionDetails.value!.domain!.useFQDN 
          ? `https://${selectionDetails.value!.domain!.customDomain}`
          : `https://${gwConfig.subdomain}.${selectionDetails.value!.domain!.selectedDomain!.publicConfig.domain}`;
```
so there's no need for an extra api call or find method, anyway it's up to you :)

Reply:
goo suggestion, thanks